target/linux/lantiq/image/Makefile

```diff
@@ -205,9 +213,10 @@ ifeq ($(SUBTARGET),xway)
 
 define Device/bt_homehub-v2b
   $(Device/NAND)
+  DEVICE_VENDOR := BT
```

Comment:
British Telecom

Reply:
@mwarning do you mind updating the script with the correct names?

Reply:
@mkresin British Telecom is the old name (https://en.wikipedia.org/wiki/BT_Group), but I have changed it for now. This may need a third opinion.